inc/search.class.php

```diff
@@ -1264,7 +1264,6 @@ static function constructData(array &$data, $onlycount = false) {
          }
       }
 
-      $data['data']['execution_time'] = $DBread->execution_time;
       if (isset($data['search']['savedsearches_id'])) {
          SavedSearch::updateExecutionTime(
```

Comment:
Maybe this one sould be updated as well only on success?

Reply:
OK for me, commit amended.

An extra commit added for a refactor I would like to do. If you think it is too risky, then I  may remove it from the PR.

Calls to Search::constructDatas() never get the returned value

![image](https://user-images.githubusercontent.com/14139801/66111288-d74edb00-e5c8-11e9-8a2a-6e3ebf68190b.png)